src/main/java/com/google/api/codegen/discovery/transformer/SampleNamer.java

```diff
@@ -38,8 +38,9 @@ public String getSampleClassName(String apiTypeName) {
   /**
    * Returns the variable name of the service.
    */
-  public String getServiceVarName(String apiTypeName) {
-    return localVarName(Name.lowerCamel(Name.upperCamel(apiTypeName).toLowerCamel(), "service"));
+  public String getServiceVarName(String lowerCamelApiTypeName) {
+    return localVarName(
+        Name.lowerCamel(Name.lowerCamel(lowerCamelApiTypeName).toLowerCamel(), "service"));
```

Comment:
The reason I included lowerCamelApiTypeName was for cases where the capitalizations were wrong:

`SQLAdmin` -> `sQLAdmin`

I don't think that that can be fixed without some manual override, but this line can be shortened still, so I'll use the code you provided. Thanks!